### PR TITLE
JBTIS-871 - add estimated size support for additional features

### PR DESCRIPTION
**Pull request by pleacu**
Also corrected the column label to 'Feature' and bumped the pixel hight by 5 to avoid unnecessary scrollbars.

@dgolovin - please review - thkx!

### CODE REVIEW

File: installer/src/panels/com/jboss/devstudio/core/installer/bean/P2IU.java [hunk 1]
```diff
@@ -14,16 +14,18 @@
 	private String description;
 	private Boolean selected;
 	private String path;
+	private String size;  // estimated size - needed only in summary panel
 	
-	public P2IU(Boolean selected, String id, String label, String description, String path) {
+	public P2IU(Boolean selected, String id, String label, String description, String path, String size) {
```
Comment (dgolovin) [inline installer/src/panels/com/jboss/devstudio/core/installer/bean/P2IU.java]: This should break tests code here https://github.com/jbdevstudio/jbdevstudio-product/tree/master/installer/src/test/com/jboss/devstudio/core/installer. This is just my guess, I did not really try it in eclipse. Tests probably are not not compiled during the build and not executed :)

File: installer/src/panels/com/jboss/devstudio/core/installer/bean/P2IUListBean.java [hunk 2]
```diff
@@ -89,4 +88,25 @@ public String getCommaSeparatedLocationStringList() {
 		}
 		return allLocationsList.toString();
     }
+
+    /**
+     * 
+     * @return a comma-separated list of additional IU sizes (estimated)
+     */
+    public String getCommaSeparatedSizeStringList() {
```
Comment (dgolovin) [inline installer/src/panels/com/jboss/devstudio/core/installer/bean/P2IUListBean.java]: Please add test for this method to [P2IUListBeanTest.java](https://github.com/jbdevstudio/jbdevstudio-product/blob/master/installer/src/test/com/jboss/devstudio/core/installer/bean/P2IUListBeanTest.java).

File: installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java [hunk 3]
```diff
@@ -66,13 +73,21 @@ private long getInstallationSize()
          Pack p = (Pack) iter.next();
          totalSize += p.nbytes;
       }
-     
+      
+      // Account for any additional features in the total aggregate size.
+      String addFeatureSizes = idata.getVariable("INSTALL_ADDITIONAL_SIZES");
```
Comment (dgolovin) [inline installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java]: This two blocks look alike. Would be good to ether concatenate sizes and then do all in on cycle or create static method for that cycle.

Comment (pleacu) [inline installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java]: @dgolovin - the testing is a bit of a mystery - please check things now

Comment (dgolovin) [inline installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java]: @pleacu I've fixed the tests and sent PR for your branch https://github.com/pleacu/jbdevstudio-product/pull/1.

Comment (dgolovin) [inline installer/src/panels/com/jboss/devstudio/core/installer/DiskSpaceCheckPanel.java]: If you want execute tests there should be launch item available already in toolbar:
image
or in Run Configuration Dialog:
image

**Comment by pleacu** [comment]
Thanks Denis - I've merged your change into my branch for the PR

**Comment by pleacu** [comment]
@alexeykazakov - please push at your convenience.

**Pull request merged**